include/nlohmann/detail/macro_scope.hpp

```diff
@@ -292,6 +293,41 @@
     inline void to_json(nlohmann::json& nlohmann_json_j, const Type& nlohmann_json_t) { NLOHMANN_JSON_EXPAND(NLOHMANN_JSON_PASTE(NLOHMANN_JSON_TO, __VA_ARGS__)) } \
     inline void from_json(const nlohmann::json& nlohmann_json_j, Type& nlohmann_json_t) { NLOHMANN_JSON_EXPAND(NLOHMANN_JSON_PASTE(NLOHMANN_JSON_FROM, __VA_ARGS__)) }
 
+
+// source: https://stackoverflow.com/a/26745591
```

Comment:
Please do not use code from StackOverflow - its license (CC BY-SA 3.0) is incompatible to MIT.

Reply:
The code is inspired from SO, but is completely different.

Reply:
Then please still rephrase the comment - I had issues in the past that automatic license checking tools raised their concerns which blocked people from using the company until they could convince their lawyers otherwise...

Reply:
Is it fine if I drop it?

Reply:
reworded instead

Reply:
Thanks!